modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -1100,6 +1100,8 @@ iocshBody (const char *pathname, const char *commandLine, const char *macros)
 
     macPushScope(handle);
     macInstallMacros(handle, defines);
+     if (pathname)
```

Comment:
Somehow the indentation is 5 spaces on this line.  Please change to 4.  (the next line has 8, which is correct)